extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnPageReaderImpl.java

```diff
@@ -64,6 +63,7 @@ public class ColumnPageReaderImpl implements ColumnPageReader {
     private PageHeader pageHeader;
     private int numValues;
     private int rowCount = -1;
+    private SeekableChannelsProvider.ChannelContext context;
```

Comment:
I think this is probably wrong. We probably want the actual methods that use a context to have a signature change to accept one. If we do it this way, the context becomes "persistent", rather than being associated with a particular operational step.